style.css

```diff
@@ -0,0 +1,408 @@
+/*font-family: 'Oswald', sans-serif;
+font-family: 'Lato', sans-serif;
+font-family: 'Merriweather', serif;
+font-family: 'Lora', serif;
+light gray color: #D7D8C5;*/
+
+body {
+  padding: 0;
+  margin: 0;
+}
+/*top of page*/
+.cssoff {
+  font-family: 'Lato', sans-serif;
+  font-weight: 100;
+  font-size: 16em;
+  padding-top: 0;
+  padding-left: .5em;
+  color: #FFFFFF;
+  z-index: 200;
+  position: absolute;
+}
+.purple-rectangle {
```

Comment:
Naming is difficult, but we need to push ourselves past the obvious and look into possibilities of abstractions.